libs/database/src/lib/schemas/language.schema.ts

```diff
@@ -21,11 +21,10 @@
  * or have any questions.
  */
 import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
-import { LanguagePayloadDto } from '../dtos/language.dto';
 import { CastcleBase } from './base.schema';
 
 @Schema({ timestamps: true })
-class LanguageDocument extends CastcleBase {
+export class Language extends CastcleBase {
```

Comment:
Remove all `@Prop()` and `@Schema` from mongoose.
Use plain class and move this class to `language.model.ts`.
Remove unused `LanguageSchema`

- Country
- Language
- ReportingSubject